onnxruntime/core/providers/cann/cann_execution_provider.cc

```diff
@@ -1455,6 +1455,7 @@ void CANNExecutionProvider::RegisterAllocator(AllocatorManager& allocator_manage
                                                         static_cast<int>(info_.arena_extend_strategy),
                                                         -1,
                                                         -1,
+                                                        -1,
                                                         -1)},
```

Comment:
last one is int64:  -1L

Reply:
the conversion should be fine from int to int64_t

Reply:
I'll mark this as resolved - let me know if you want me to update it